### PR TITLE
Use standard for linting

### PR DESCRIPTION
**Pull request by hiimbex**
Re: https://github.com/probot/probot/issues/235

### CODE REVIEW

**Comment by hiimbex** [comment]
So on this branch for whatever reason I didn't have to add `eslint-plugin-import` re: https://github.com/probot/github-app/pull/9 @gr2m @bkeepers and locally that's still true for me. Just curious if this branch passes tests locally for the both of you?

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "main": "index.js",
   "scripts": {
     "start": "probot run ./index.js",
-    "test": "mocha && xo"
+    "test": "mocha && standard"
```
Comment (gr2m) [inline package.json]: I usually put "standard" in a `"pretest"` script, like here:
https://github.com/hoodiehq/hoodie/blob/9bcd805fd01e905dd53b94c5ca547967cdd3f893/package.json#L73

Comment (hiimbex) [reply package.json]: I didn't know npm had a `pretest` field! cool!

Comment (gr2m) [reply package.json]: oh yeah, you can use `pre<script name>` and `post<script name>` for any of the scripts defined in package.json, see https://docs.npmjs.com/misc/scripts

> Pre and post commands with matching names will be run for those as well (e.g. premyscript, myscript, postmyscript)

Comment (bkeepers) [reply package.json]: My preference would be posttest. I would rather focus on getting the tests passing before worrying about the aesthetics of the code.

**Pull request merged**